src/admin-views/meta-box.php

```diff
@@ -16,6 +16,10 @@
 $attendees_url = Tribe__Tickets__Tickets_Handler::instance()->get_attendee_report_link( get_post( $post_id ) );
 ?>
 
+<div class="tribe-tickets-editor-block">
```

Comment:
Would it be better if this was `tribe-tickets-editor-blocker` as "block" is pretty generic and sounds like a container/wrapper?